code/src/components/app-badge/AppBadge.tsx

```diff
@@ -0,0 +1,31 @@
+import Badge from "@mui/material/Badge";
+
+import { AppBadgeProps } from "@/components/app-badge/AppBadge.types";
+
+import cn from "@/utils/cn";
+
+import "@/components/app-badge/AppBadge.scss";
+
+const AppBadge = ({
+  className,
+  isRounded = true,
+  size = "medium",
+  color = "contained",
+  ...props
+}: AppBadgeProps) => {
+  return (
+    <Badge
+      max={10}
```

Comment:
why are you decided used `max` with `10` value here ?

Reply:
I deleted it. And if we need this max value, we can pass it like props